### ISSUE TITLE
Add popover button for web map preview

### ISSUE THREAD

**Issue by cole-fields** [genkimaps/gis-hub#122]
Add a map location button on /datasets page that will open a bootstrap popover with a web map. Map will contain the spatial field from the metadata (unioned convex hull geometry).

**Comment by cole-fields** [comment]
Known problem: `Map container is already initialized` when cycling between map popovers. if popover for a dataset is closed by clicking it's own marker button, error will not occur. if closed by clicking another dataset's marker location, error will occur when user re-clicks original marker button. I've tried debugging this extensively, but no solution yet. Frustrating, but not the end of the world for now.

**Comment by cole-fields** [comment]
@genkimaps Just a note, that I have the gishub server running on the pmap branch for the moment. This branch has this popover element added. There was a bug where the popover maps were preventing the spatial filter leaflet map from working. I changed the popover map to use OpenLayers instead of Leaflet and this resolved the issue.

**Comment by cole-fields** [comment]
@genkimaps You were right about the `div id="map"` even when using OpenLayers. I ended up creating a unique id for the div and setting it up so that the current map closes when the user clicks the location button for a different dataset. Before, it would sometimes render a blank popover page since the map container was already initialized. I've merged this with master and have the gishub server back on that now.
